apps/component-date/index.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const summary = require('hof-behaviour-summary-page');
+
+module.exports = {
+  name: 'component-date',
+  // this will mean you will need to go to http://localhost:8080/component-date
+  baseUrl: '/component-date',
+  steps: {
+    '/date-examples': {
```

Comment:
perhaps split these over two steps, and don't use the word 'examples' in the paths?